src/renderer.ts

```diff
@@ -415,7 +416,7 @@ const captionElement = (format: Format) => (
 				children,
 			);
 		case 'A':
-			return h(
+			return styledH(
```

Comment:
`React.createElement` doesn't support emotion's `css` the way JSX does. If you look at an apps-rendering gallery at the moment for example you'd see that the dom element doesn't have the correct styles and instead has a `css=[Object object]` attribute. This should fix that.

Reply:
Great, thanks for the explainer!

Reply:
Emotion takes over in the JSX-to-createElement pipeline so that it can handle the `css` prop before handing over to `React.createElement` (seen [here](https://github.com/guardian/apps-rendering/blob/5935a859732ecd4824827d1e4c9596d57bd8b8ce/webpack.config.ts#L101) in the config, which uses the [React 17 JSX transform](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html)).